packages/story-editor/src/components/inspector/inspectorProvider.js

```diff
@@ -36,11 +36,12 @@ import { useFeature } from 'flagged';
 import { useAPI } from '../../app/api';
 import { useStory } from '../../app/story';
 import { useHighlights } from '../../app/highlights';
-import { DOCUMENT, DESIGN, PUBLISH_MODAL_DOCUMENT } from './constants';
+import { DOCUMENT, STYLE, PUBLISH_MODAL_DOCUMENT, INSERT } from './constants';
 import Context from './context';
 import DesignInspector from './design';
```

Comment:
This should probably also be renamed _Style_ now?

Reply:
I'm +1 with Miina [here](https://github.com/GoogleForCreators/web-stories-wp/pull/10760#discussion_r817626896), I guess you too? So leaving it as-is for now.